test/logger_test.rb

```diff
@@ -217,7 +217,7 @@
       assert_equal 'Artificially raised exception', actual_log[:message]
       assert_equal 'divided by 0', actual_log[:error][:message]
       assert_equal 'ZeroDivisionError', actual_log[:error][:type]
-      assert_match 'logger_test.rb', actual_log[:error][:stack_trace].lines.first
+      assert_match 'logger_test.rb', actual_log[:error][:stack_trace].first
```

Comment:
I think we should check the whole array rather than just the first element to ensure its working as we intended

Reply:
I worry that if we specify too much of the stack, we will make this test very brittle.  However i will add the full relative path of the file.

Reply:
Yeah that make sense I forgot that trace also have time and other traces.